subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/artifact_transforms.adoc

```diff
@@ -95,6 +95,7 @@ include::sample[dir="snippets/dependencyManagement/artifactTransforms-minify/gro
 <2> All JAR files are not minified
 <3> Request `minified=true` on all resolvable configurations
 <4> Add the dependencies which will be transformed
+<5> Create Task to execute the artifact transformation
```

Comment:
💄 The rest of the bullet points use "Add" as the verb, I would do the same here for consistency.